### PR TITLE
e2e mla

### LINKED ISSUE #3388: [E2E] MLA
Add e2e tests for mla

### PR DESCRIPTION
**Pull request by kgroschoff**
### What this PR does / why we need it
Adds e2e tests for mla

### Release note

```release-note
NONE
```

### CODE REVIEW

File: cypress/integration/stories/mla.spec.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import {WizardPage} from '../../pages/wizard.po';
+import {login, logout} from '../../utils/auth';
+import {Condition} from '../../utils/condition';
+import {Preset} from '../../utils/preset';
+import {Datacenter, Provider} from '../../utils/provider';
+import {View} from '../../utils/view';
+import {WizardStep} from '../../utils/wizard';
+import * as _ from 'lodash';
+
+xdescribe('MLA Story', () => {
```
Comment (maciaszczykm) [inline cypress/integration/stories/mla.spec.ts:23]: I see that it's in review. Is this turned off for some reason? Besides that the test code looks fine, we can merge once the tests will pass.

Comment (kgroschoff) [reply cypress/integration/stories/mla.spec.ts:23]: no that was not intended 😅 
However, it seems that I again need to take a look into the ci kubermatic configuration, as I'm currently getting `make: *** [Makefile:63: run-e2e-ci] Error 1` 🤔

Comment (maciaszczykm) [reply cypress/integration/stories/mla.spec.ts]: Sure, ping me for review/approval.

**Pull request closed without merging**